src/app/components/Banner/Banner.js

```diff
@@ -0,0 +1,19 @@
+import cn from './Banner.module.scss';
+
+const APPLICATION_FORM = 'https://forms.gle/e8zAUSGniWopQDNP9';
+
+function Banner() {
+	return (
+		<div className={cn.banner}>
+			<p>
+				Interested in becoming part of Design at UCI&apos;s Board? Submit a{' '}
+				<a className='color ultraviolet' href={APPLICATION_FORM}>
```

Comment:
can we make this open in a new tab instead?

Reply:
Addressed in 7bc6df6